spec/no_duplicate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+git clone https://github.com/puppetlabs/puppetlabs-stdlib
+
+stdlib_funcs=`ls puppetlabs-stdlib/lib/puppet/parser/functions`
+extlib_funcs=`ls lib/puppet/parser/functions`
```

Comment:
[no](http://blag.esotericsystems.at/igor/says/no)
don't do that. http://mywiki.wooledge.org/ParsingLs